### DISCUSSION TITLE
cf-convention information and opportunity to present at ESIP

### DISCUSSION THREAD

**Discussion by semantic-nomad** [cf-convention/discuss#339] [Q&A about using CF] [answered]
Hi, 
Apologies in advance if this is not the proper procedure to interact with the community. It seemed from the text on the [cfconventions.org](https://cfconventions.org/) page this avenue is preferred; it seemed this would be more appropriate as an issue rather than a discussion.
 
I am organizing a session at the upcoming ESIP July meeting focusing on harmonizing the semantic artifacts relevant to Earth, environmental and space applications (broadly considered). The short version is, there is quite a lot of resources/vocabularies in use and/or under development.  In this session we hope to dig a little deeper into what is available and how, or if, there might be ways to coordinate and collaborate.

With that, is there someone who is willing and able to provide information about the CF metadata conventions?  At this stage, any introductory information would be useful but there would be particularly interested in finding out more about the following:

1) Why was the resource developed?
2) How is the resource used?
3) Is the thesaurus currently under development?
4) What is the governance model for the resource (how are decisions made)?
5) What tooling is used to develop and publish the resource?
6) Is there any planned future development?

The meeting is will be held in Charlotte, North Carolina, July 22-26th. Remote participation is available.

I am happy to discuss further as needed. If there are any questions or concerns please do let me know.

Thank you in advance for your time.
Cheers,
/Brandon

**Comment by ethanrd** [comment, marked as answer]
Hi Brandon,

The CF Metadata Conventions are under active development and the governance model is described [here](https://cfconventions.org/governance.html).

I will be at the ESIP meeting in two weeks. Happy to talk more here and during ESIP.

Ethan
